pipeline_dp/dp_computations.py

```diff
@@ -435,3 +436,30 @@ def compute_dp_var(count: int, normalized_sum: float,
         dp_mean += compute_middle(dp_params.min_value, dp_params.max_value)
 
     return dp_count, dp_mean * dp_count, dp_mean, dp_var
+
+
+def compute_dp_count_noise_params(
+        dp_params: MeanVarParams) -> Tuple[str, float]:
```

Comment:
Should we return the enum instead of str?

Reply:
good point! Starting from that, I've rewritten this function only to return std :). It will provide simpler API and anyway noise_kind (as enum) as a  property of `dp_params`, so the caller of this function it's trivial to get `noise_kind`